code/media/src/main/java/com/adobe/marketing/mobile/media/internal/MediaOfflineService.java

```diff
@@ -341,15 +341,17 @@ boolean reportCompletedSessions() {
                                     Log.debug(
                                             MediaInternalConstants.EXTENSION_LOG_TAG,
                                             LOG_TAG,
-                                            "ReportCompletedSessions - Http request error,"
-                                                    + " connection was null");
+                                            "reportCompletedSessions - Failed to report session %s"
+                                                    + " because the connection is null (network is"
+                                                    + " offline).",
+                                            currentReportingSession);
                                 } else {
```

Comment:
can we also move the connection close L388-390 at the end of this block to avoid an extra check?